node_modules/oae-core/linkpreview/js/linkpreview.js

```diff
@@ -135,8 +137,11 @@ define(['jquery', 'oae.core', 'jquery.oembed'], function($, oae) {
                     embeddable = false;
                 }
             }
+            // Only embed text, html, and types that the browser has plugins to support
+            var supportedTypes = _.union(_.pluck(navigator.mimeTypes, 'type'), ['text/html', 'text/plain']);
+            var supported = _.contains(supportedTypes, widgetData.previews.targetType.split(';')[0]);
```

Comment:
We would definitely re-preview-process all links, but `targetType` could still be `null`. For example, the preview processor could not have finished processing the link yet, or the preview processor was not able to connect to the server. In that case, `widgetData.previews.targetType.split` would cause a JS error.

I'm not entirely sure if we should assume that the link is supported when `targetType` is null. It seems like the most sensible approach, but maybe assuming that it isn't supported would be safer?